### ISSUE TITLE
Snippets (ultisnips) not showing up in dialog

### ISSUE THREAD

**Issue by andoni0305** [ChristianChiarulli/nvim#121]
Hey!
Love the config and tutorials - they are really helpful... thanks for those
I guess this is more of a doubt than an issue

Regarding snippets; i am following your changes from coc into lsp but i can't manage to get the snippets to show in the dialog when on lsp. Is there additional changes to get those working or do we need to switch a different plug for them to work with lsp config?

Appreciate the help

**Comment by ChristianChiarulli** [comment]
Yes, I have yet to get around to implementing snippets for the new stuff, I'll leave this open until I do.

**Comment by andoni0305** [comment]
Thanks!
If i happen to get that configured beforehand I'll issue an mr with my findings

**Comment by ChristianChiarulli** [comment]
That would be great thanks!

**Comment by andoni0305** [comment]
hi @ChristianChiarulli 
so it turns out it's not too hard to get the snippets working

in the `lua/plugins/compe-config.lua` we were just missing to pass the `ultisnips` options with `true` inside the `source` object

here is how it ended up

```lua
-- TODO we need snippet support and to maybe get better docs idk

vim.cmd [[set shortmess+=c]]
vim.o.completeopt = "menuone,noselect"

require'compe'.setup {
  enabled = true;
  autocomplete = true;
  debug = false;
  min_length = 1;
  preselect = 'enable';
  throttle_time = 80;
  source_timeout = 200;
  incomplete_delay = 400;
  max_abbr_width = 100;
  max_kind_width = 100;
  max_menu_width = 100;
  documentation = true;

  source = {
    path = true;
    buffer = true;
    calc = true;
    -- vsnip = true;
    nvim_lsp = true;
    nvim_lua = true;
    spell = true;
    tags = true;
    snippets_nvim = true;
    ultisnips = true;     --  <========= We were missing this
    treesitter = true;
  };
}
```

you get the ultisnips snippets showing with that

but i am still trying to figure out how to get the documentation showing like the one with lsp-saga

but some progress at least

**Comment by wwwdata** [comment]
Hi, I also want to use Ultisnips because I was using that before switching over to lunarvim. How do I integrate it with all the latest changes into the auto-complete suggestions?
